src/test/java/com/google/javascript/gents/singleTests/enum.ts

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * A non-trivial comment.
+ * @enum {number}
+ */
+export enum NumEnum {
+  A,
+  B
+}
+
+/** @enum {number} */
```

Comment:
It'd nice if we can remove the Closure annotations for converted enums.

Reply:
Yes, see my commit comment. It appears to be all or nothing decision and I went for nothing, because of the cases not coverred. Maybe we can refactor gents to keep track of what was converted. In anycase that would require some heavier changes, maybe we can pair on that on Monday.